performance-tests/bench/udp_latency/udp_latency.cpp

```diff
@@ -0,0 +1,241 @@
+  }
+
+  cli_dgram.send (&data, 0, server_addr);
+
+  return 0;
+}
+
+static void
+receiver(ACE_SOCK_Dgram& cli_dgram)
+{
+  ACE_INET_Addr peer_addr;
+  DataSample data;
+  ACE_Time_Value timeout(2,0);
+  ssize_t rcv_cnt = 1;
+
+  Bench::WorkerReport report{};
+  report.process_report.participants.length(1);
+  report.process_report.participants[0].subscribers.length(1);
+  report.process_report.participants[0].subscribers[0].datareaders.length(1);
+  Builder::PropertySeq& properties = report.process_report.participants[0].subscribers[0].datareaders[0].properties;
+
+  const size_t buffer_size = static_cast<size_t>(1.1 * frequency * test_duration);
+
+  auto round_trip_latency_stat_block = std::make_shared<Bench::PropertyStatBlock>(properties, "round_trip_latency", buffer_size);
+
+  while (rcv_cnt) {
+    rcv_cnt = cli_dgram.recv (&data,
+                              sizeof (data),
+                              peer_addr,
+                              0,
+                              &timeout);
+    if (rcv_cnt > 0) {
+      round_trip_latency_stat_block->update(5E-10 * (ACE_OS::gethrtime() - data.sending_time_));
+    }
+    else if (rcv_cnt < 0) {
+      ACE_ERROR((LM_ERROR,
+                 ACE_TEXT("(%P|%t) %p\n"),
+                 ACE_TEXT("cli_dgram.recv")));
+      return;
+    }
+  }
+
+  round_trip_latency_stat_block->finalize();
+
+  if (!report_file_name.empty()) {
+    ofstream report_file(report_file_name);
+    if (report_file.good()) {
+      idl_2_json(report, report_file, 9u);
+    }
+  }
+
+  return;
+}
+
+int echo_server(int port)
+{
+
+  ACE_INET_Addr server_addr;
+  server_addr.set (port);
+  ACE_SOCK_Dgram server_dgram;
+
+  if (server_dgram.open (server_addr) == -1)
+  {
+    ACE_ERROR((LM_ERROR,
+               ACE_TEXT("(%P|%t) %p\n"),
+               ACE_TEXT("server_dgram.open")));
+    return 1;
+  }
+
+  ACE_INET_Addr peer_addr;
+  DataSample data;
+  ACE_Time_Value timeout(120,0);
+  ssize_t rcv_cnt = 1;
+
+  while (rcv_cnt) {
+    rcv_cnt = server_dgram.recv (&data,
+                              sizeof (data),
+                              peer_addr,
+                              0,
+                              &timeout);
+    if (rcv_cnt >= 0) {
+      if (server_dgram.send(&data,
+                            rcv_cnt,
+                            peer_addr) < 0){
+        ACE_ERROR((LM_ERROR,
+                   ACE_TEXT("(%P|%t) %p\n"),
+                   ACE_TEXT("server_dgram.send")));
+        return 1;
+      }
+    }
+    else if (rcv_cnt < 0) {
+      ACE_ERROR((LM_ERROR,
+                 ACE_TEXT("(%P|%t) %p\n"),
+                 ACE_TEXT("server_dgram.recv")));
+      return 1;
+    }
+  }
+  return 0;
+}
+
+int parse_args(int argc, ACE_TCHAR ** argv)
+{
+  int c;
+
+  ACE_Get_Opt getopt(argc, argv, "c:d:f:m:s:r:");
+  bool ok = true;
+
+  while (ok && (c = getopt()) != -1) {
+    switch (c) {
+      case 'c':
+        ok = (server_addr.set(getopt.opt_arg()) == 0);
+        break;
```

Comment:
```suggestion
    switch (c) {
    case 'c':
      ok = (server_addr.set(getopt.opt_arg()) == 0);
      break;
```

For other cases as well